packages/flutter_tools/pubspec.yaml

```diff
@@ -110,6 +110,10 @@ dev_dependencies:
   pubspec_parse: 1.2.3
 
   checked_yaml: 2.0.3 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  # ffi is not used directly by flutter_tools, but including it here is the most convenient way to make the
+  # --experimental-faster-testing option of flutter test work. See _generateIsolateSpawningTesterPackageConfig
+  # in lib/src/test/runner.dart for more details.
```

Comment:
I don't think this note is quite accurate - flutter_tools will generate code in some cases that uses this dependency. 

I'm not sure how important it is to document this case - more might get added later even if this one gets removed. As long as a test would fail by removing this (which, it should right?) then let's just remove these comments.

Reply:
Done.